lib/src/crypto/pem.dart

```diff
@@ -87,13 +87,14 @@ RSAPrivateKey _extractRSAKeyFromDERBytes(Uint8List bytes) {
     if (asn is ASN1Sequence) {
       var objects = asn.objects;
       if (objects.length == 3 && objects[2] is ASN1OctetString) {
-        ASN1OctetString string = objects[2];
+        ASN1OctetString string = objects[2] as ASN1OctetString;
```

Comment:
[nit] Prefer to fix the type on the left when introducing the explicit cast rather than in a followup.

```suggestion
        var string = objects[2] as ASN1OctetString;
```

Reply:
Done, thanks.